Automattic/vip-go-mu-plugins
# Add Jetpack 11.2 subtree

## Description
This change adds the Jetpack 11.2 subtree to mu-plugins and thusly introduces the Jetpack 11.2 Beta to users of WPVIP.

## Changelog Description
### Plugin Updated: Jetpack 11.2

The Beta release of Jetpack 11.2 has been added to mu-plugins. Users may test the beta by [pinning their version of Jetpack](https://docs.wpvip.com/how-tos/test-jetpack-on-vip-go/#h-pinning-to-a-version) to version 11.2 in VIP.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1. Check out PR.
2. Pin Jetpack to version 11.2
1. Go to `wp-admin` > `Jetpack`
1. Verify version is 11.2.
